src/cogent3/core/new_alignment.py

```diff
@@ -198,10 +196,16 @@ def get_seq_bytes(
 
     def get_seq_view(self, seqid: str) -> SeqDataView:
         seq_len = len(self._data[seqid])
-        return SeqDataView(seq=self, seqid=seqid, seq_len=seq_len)
+        return SeqDataView(seqs=self, seqid=seqid, seq_len=seq_len)
+
+    def subset(self, names: Union[str, typing.Sequence[str]]) -> SeqsData:
```

Comment:
Should creating a new `SeqsData` happen without getting a `SeqDataView` first? i.e. when should `subset` be used instead of `__getitem__`?

Reply:
If you want to subset your data (to a subset of more than one), I don't think it makes sense to go via individual views